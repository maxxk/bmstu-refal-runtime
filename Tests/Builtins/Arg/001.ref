$ENTRY Go
{
	= <Prout <Arg 1>>
	  <Prout <Arg 2>>
	  <Prout <Arg 3>>
	  <Prout <Arg 4>>
	  <Prout <Arg 5>>
}


