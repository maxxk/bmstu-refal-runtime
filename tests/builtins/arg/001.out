Абракадабра
тест
asdsd
123123
sdfqwe
